mfr/extensions/pdf/exceptions.py

```diff
@@ -0,0 +1,23 @@
+from mfr.core.exceptions import ExporterError
+
+
+class PillowImageError(ExporterError):
```

Comment:
Seems duplicated, but I guess you have a good reason for not using `mfr.extensions.image.exceptions.PillowImageError`?

Reply:
It is duplicated, but it doesnt make sense to import an exception from a different extension (for the most part, extensions dont interact much) thats why its copied this way. Thoughts?

Reply:
I see what you mean and I agree. Let's leave this here for @felliott in Phase 2 Review.

Reply:
I'm okay with duplicating this, but the docstrings and `__TYPE` should be consistent with the new location.  `__TYPE` should probably be `pdf_pillow`.